### PR TITLE
chore: Set Dependabot strategy to 'increse' 

### PR DESCRIPTION
**Pull request by yyvess**
* Set Dependabot strategy to 'increse'
* Update version semantic to patch

Note :
The default strategy of Dependabot is to update only the package-lock.
I change the strategy to **increase** to update the package.json
I update version semantic to patch as Dependabot will update the package.json.
PS. 'Increase' strategy don't care about version constraints on the package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,13 +26,13 @@
...
-    "jest": "^26.4.2",
-    "playwright": "^1.4.2",
-    "prettier": "^2.1.2",
-    "promise": "^8.1.0"
+    "eslint": "~7.14.0",
+    "eslint-config-prettier": "~6.15.0",
+    "eslint-plugin-prettier": "~3.1.4",
+    "express": "~4.17.1",
+    "jest": "~26.6.3",
+    "playwright": "~1.6.2",
```
Comment (jfgreffier) [inline package.json:34]: Can you set it to `~1.4.2`, do a npm install and commit the package-lock.json ? In this way, we can verify that dependabot performs the update on next run

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,7 @@
-node_modules
-coverage
+node_modules/
+coverage/
 .DS_Store
 yarn.lock
 .vscode
+.idea/
+*.iml
```
Comment (jfgreffier) [inline .gitignore]: Minor : check if you set up editorconfig support; it's a plugin in intellij if I remember. According to `.editorconfig`, files end with a newline

**Review by jfgreffier — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (yyvess) [reply package.json:34]: I was tested it my fork it's look good =>
image
PS: I change the default branch and force to run Dependabot manually to test it

**Review by yyvess — COMMENTED** [review]
I update my config 😉

**Pull request merged**